docs/plugins/vertex-ai.md

```diff
@@ -96,6 +96,30 @@ const llmResponse = await generate({
 });
 ```
 
+This plugin also supports grounding Gemini text responses using
+[Google Search](https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/ground-gemini#web-ground-gemini)
```

Comment:
This is not a cheap feature. Wonder if it's worth linking to pricing....

Reply:
Yeah I think it's important to add a highlighted note with the link, do you know if the following syntax can show as a note in the [Firebase page docs](https://firebase.google.com/docs/genkit/plugins/vertex-ai)? I think the docs there is fetched from the markdown files here?
> [!IMPORTANT]
> Please beware of the costs associated with groudnding responses with Google Search, check the [pricing](https://cloud.google.com/vertex-ai/generative-ai/pricing#gemini-models) page for more information.

Reply:
Yes, something like this LGTM. Added @kevinthecheung to comment on the wording.
/cc @chrisraygill

Reply:
I suggest:

```suggestion
Important: Vertex AI charges a fee for grounding requests in addition to the cost of making LLM requests.  See the [Vertex AI pricing](link) page and be sure you understand grounding request pricing before you use this feature.
```

DevSite will render it as a note